include/readout/concepts/RawDataProcessorConcept.hpp

```diff
@@ -28,15 +28,15 @@ class RawDataProcessorConcept
     delete; ///< RawDataProcessorConcept is not move-assignable
 
   virtual void conf(const nlohmann::json& cfg) { set_emulator_mode(cfg.get<datalinkhandler::Conf>().emulator_mode); }
-  std::uint64_t get_last_daq_time() { return m_last_processed_daq_ts.load(); }
+  std::uint64_t get_last_daq_time() { return m_last_processed_daq_ts.load(); } // NOLINT(build/unsigned)
```

Comment:
Especially due to the trailing implementations, I would prefer to have spaces and comments between the functions in form of doxy oneliner comments.